ui/src/components/SummaryCards/ParcelSizeSummaryCard.jsx

```diff
@@ -9,7 +9,7 @@ export default function ParcelSizeSummaryCard(props) {
     <div className="form-wrapper summary-subform-wrapper package-type-summary-wrapper">
       <div className="form-inner form-inner-summary">
         <div className="form-header summary-subheader">
-          {props.selectedPackage.name}
+          {props.selectedPackageSize.packageSize.title}
```

Comment:
![image](https://user-images.githubusercontent.com/55206909/115995090-89eaf200-a5e2-11eb-816d-99896588e58f.png)
I am getting this error when trying to open order modal. Passing only package size should be enough here (not whole package option).

Reply:
Do you have a seed for testing orders?

Reply:
I just send POST requests with Postman